### Question 3
<p>
    answers:
</p>
<p>
    a: iii, men will earn more than felames
</p>
<p>
    b: 
</p>

In [3]:
#B
partb_answer<-50 + 20 * 4.0 + 0.07 * 110 + 35 * 1 + 0.01 * 4.0 * 110 - 10 * 4.0 * 1
partb_answer

[1] 137.1

<p>
    C: False , Coeficient value depends on the scale of associate variables.
</p>

### Question 10


In [5]:
#part a
#install.packages('ISLR')
library(ISLR)
Carseats<-Carseats
str(Carseats)

'data.frame':	400 obs. of  11 variables:
 $ Sales      : num  9.5 11.22 10.06 7.4 4.15 ...
 $ CompPrice  : num  138 111 113 117 141 124 115 136 132 132 ...
 $ Income     : num  73 48 35 100 64 113 105 81 110 113 ...
 $ Advertising: num  11 16 10 4 3 13 0 15 0 0 ...
 $ Population : num  276 260 269 466 340 501 45 425 108 131 ...
 $ Price      : num  120 83 80 97 128 72 108 120 124 124 ...
 $ ShelveLoc  : Factor w/ 3 levels "Bad","Good","Medium": 1 2 3 3 1 1 3 2 3 3 ...
 $ Age        : num  42 65 59 55 38 78 71 67 76 76 ...
 $ Education  : num  17 10 12 14 13 16 15 10 10 17 ...
 $ Urban      : Factor w/ 2 levels "No","Yes": 2 2 2 2 2 1 2 2 1 1 ...
 $ US         : Factor w/ 2 levels "No","Yes": 2 2 2 2 1 2 1 2 1 2 ...


<p>Sales:
Unit sales (in thousands) at each location
</p>
<p>CompPrice:
Price charged by competitor at each location

</p>
<p>Income: 
Community income level (in thousands of dollars)

</p>
<p>Advertising: 
Local advertising budget for company at each location (in thousands of dollars)

</p>
<p>Population: 
Population size in region (in thousands)

</p>
<p>Price: 
Price company charges for car seats at each site

</p>
<p>ShelveLoc: 
A factor with levels Bad, Good and Medium indicating the quality of the shelving location for the car seats at each site

</p>
<p>Age: 
Average age of the local population

</p>
<p>Education: 
Education level at each location

</p>
<p>Urban: 
A factor with levels No and Yes to indicate whether the store is in an urban or rural location

</p>
<p>US: 
A factor with levels No and Yes to indicate whether the store is in the US or not</p>

<p><a href="https://rdrr.io/cran/ISLR/man/Carseats.html">Source</a></p>

In [7]:
# part a
R <- lm(Sales ~ Price + Urban + US, data=Carseats)

In [8]:
# part b
summary(R)


Call:
lm(formula = Sales ~ Price + Urban + US, data = Carseats)

Residuals:
    Min      1Q  Median      3Q     Max 
-6.9206 -1.6220 -0.0564  1.5786  7.0581 

Coefficients:
             Estimate Std. Error t value Pr(>|t|)    
(Intercept) 13.043469   0.651012  20.036  < 2e-16 ***
Price       -0.054459   0.005242 -10.389  < 2e-16 ***
UrbanYes    -0.021916   0.271650  -0.081    0.936    
USYes        1.200573   0.259042   4.635 4.86e-06 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 2.472 on 396 degrees of freedom
Multiple R-squared:  0.2393,	Adjusted R-squared:  0.2335 
F-statistic: 41.52 on 3 and 396 DF,  p-value: < 2.2e-16


##### part c
<p>Sale = 13.043 - 0.0544 * Price - 0.0219 * UrbanYes + 1.2 * USYes</p>

<p> if the value of urban yes  and/or usyes  is no then these two will be zero</p>

##### part d
<p>Based on the P value null hypotesise will be rejected</p>